eventservice/events.yaml

```
@@ -872,6 +872,16 @@ components:
           type: string
           format: uuid
           description: For an event created by a Rokwire group, this field stores the group ID.
+        groupIds:
+          type: array
+          description: The groupIds is a list of all groups that a user event is associated with.
+          items:
+            type: string
+        userIds:
+          type: array
+          description: The userIds is a list of account IDs to handle "sub-groups" of users that are allowed to see a given event.
```

```suggestion
          description: A list of account IDs of users who can view this event. This field gives event view permissions to specific users within a Rokwire group.
```
Suggesting an alternative description without using the "sub-groups" term and clarifying the purpose. Thanks.